qemu/tests/numa_stress.py

```diff
@@ -79,6 +79,9 @@ def run(test, params, env):
     memory_file = utils_misc.get_path(tmpfs_path, "test")
     dd_cmd = "dd if=/dev/urandom of=%s bs=1k count=%s" % (memory_file,
                                                           tmpfs_size)
+    if utils_memory.freememtotal() < tmpfs_size:
```

Comment:
I think it's better to add step for releasing mem here.
echo 3 > /proc/sys/vm/drop_caches